### ISSUE TITLE
Audit accessibility errors in Sitewide Crew products 

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9494]
## Description

As a Sitewide Crew, I want actionable metrics to help prioritize accessibility initiatives

## ACs

- [ ] we have a tool to help prioritize and track product changes  
- [ ] ideally, this audit could generate historic data, so we can track changes over time for reporting purposes

### Sources that we want to track

- by product
- by source of problem (Editor training/guidance, CMS application, FE templates/CSS, VA Design system, etc)
- modernized / non-modernized
- others TBD?

### Products within the Sitewide Crew portfolio that should be tracked

- ~Banners and alerts~ These are contained within other URLs and wil be harder to track down.
- Benefits hubs
- Campaign Landing Page
- ~Cemetery~ Not live on modernized va.gov yet... but we will want to audit accessibility issues within the [non-modernized cemeteries websites](https://www.cem.va.gov/find-cemetery/), separately. 
- ~CMS Knowledge Base~ Not Veteran facing, this should be part of #9443.
- Facility locator
- ~Facility status~ More of a feature, contained within other products.
- ~FAQs~ This is part of R&S
- Find a VA Form
- ~Media library~ Contained within other products
- ~Office~ Not live yet
- Outreach hub
- Resources and support
- ~VA services taxonomy~ Contained within other products
- VA.gov homepage
- VA.gov megamenu
- VAMC system
- ~VBA facilities~ Not live on modernized va.gov yet... but we will want to audit accessibility issues within the [non-modernized facilities websites](https://benefits.va.gov/benefits/offices.asp), separately. 
- Vet Center
- **this list is incomplete**

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [x] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
@EWashb I think I want to close this? We have other tickets (the cypress-axe issue) for testing/reporting and while it's not exactly identical to what this was asking about I think we have moved past this slightly and changed direction a bit.  Thoughts?

**Comment by EWashb** [comment]
@laflannery i think we are safe to close this with a change in scope/direction
